src/server/routes/actions.ts

```diff
@@ -55,6 +56,7 @@ router.post<unknown, DripResponse, BotRequestType>("/bot-endpoint", (req, res) =
     .catch((e) => {
       logger.error(e);
       errorCounter.plusOne("other");
+      res.send({ error: "Operation failed." });
```

Comment:
Not disclosing any more information to the external user because we don't know what `e` is, and worst case it could contain some sensitive info.